easybuild/easyconfigs/p/Python/Python-3.6.3-icc_segfault_workaround.patch

```diff
@@ -0,0 +1,58 @@
+--- ./Modules/_sha3/kcp/KeccakP-1600-opt64.c.orig	2018-02-26 16:29:31.750301802 +0000
```

Comment:
@schiotz Please include a pointer to where you got this patch above this line (line above this line are comments)